realsense_camera/cfg/f200_params.cfg

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+PACKAGE="realsense_camera"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+#             Name                               Type      Level       Description                     Default    Min     Max
+gen.add("enable_depth",                          bool_t,   0,          "Enable Depth",                 True)
```

Comment:
Did you try dynamic enable/disable color stream for F200? If its working, we should add that too.